class AccountException(Exception):
    pass


class AccountNotFoundException(AccountException):
    pass
